gensrc/script/gen_functions.py

```diff
@@ -655,8 +656,11 @@ def initialize_sub(op, return_type, arg_types):
 
     try:
         os.makedirs(BE_PATH)
-    except FileExistsError:
-        pass
+    except OSError as e:
```

Comment:
I guess this is for older python interpreters. We also need to apply the change to gensrc/script/gen_builtins_functions.py

Reply:
Oh, I miss this.